### PR TITLE
add figures, text and refs for water4energy implementation

### PR DESCRIPTION
**Pull request by scparkinson**
https://github.com/iiasa/message_doc/issues/18
#18 
Energy: add water for energy implementation @scparkinson

### CODE REVIEW

File: source/water/index.rst [hunk 1]
```diff
@@ -1,2 +1,67 @@
...
+This enables the fuel efficiency improvements included in MESSAGE to be translated into
+improvements in water intensity. Water withdrawal and consumption intensities for power plant
+cooling technologies are calibrated to the range
+reported in Meldrum et al., (2013) :cite:`meldrum_2013`. Additional parasitic electricity demands from recirculating
+and dry cooling technologies are accounted for explicitly in the electricity balance calculation. All
+other technologies follow the data reported in Fricko et al.
+(2016) :cite:`fricko_2016`. 
+
+A key feature of the implementation is the representation of power plant cooling
+technology options for individual power plant types (figure below) :cite:`parkinson_2019`.
```
Comment (khaeru) [inline source/water/index.rst]: Ditto.

File: source/water/index.rst [hunk 2]
```diff
@@ -1,2 +1,67 @@
...
+Each power plant type that requires cooling in MESSAGE 
+is connected to a corresponding cooling technology option (once-through, recirculating or
+air cooling), with the investment into and operation of the cooling technologies included in the
+optimization decision variables. This enables MESSAGE to choose the type of cooling technology
+for each power plant type and track how the operation of the cooling technologies impact water
+withdrawals, return flows, thermal pollution and parasitic electricity use.
```
Comment (khaeru) [inline source/water/index.rst]: Oxford comma? @volker-krey what's the preferred style?

File: source/water/index.rst [hunk 3]
```diff
@@ -1,2 +1,67 @@
...
+(:math:`\phi_{cool}`) based on average heat-rate (:math:`\phi_{comb}`) and heat lost to emissions 
+(:math:`\phi_{emis}`), and this data is identified from the literature :cite:`fricko_2016`.
+
+:math:`\phi_{cool} = \phi_{comb} - \phi_{emis} - 1`
+
+With time-varying heat-rates (i.e., :math:`t =0,1,2,...`) and a constant share of energy to emissions and electricity:
+
+:math:`\phi_{cool}[t] = \phi_{comb}[t] \cdot \left( \, 1 - \dfrac{\phi_{emis}}{\phi_{comb}[0]} \, \right) - 1`
+
+Increased fuel efficiency (lower heat-rate) reduces the per unit electricity cooling requirement.
```
Comment (khaeru) [inline source/water/index.rst]: Would it be correct to write “…reduces the cooling water requirement per unit of electricity generated.” ? That would be easier to read.

Comment (scparkinson) [reply source/water/index.rst]: Thanks for the review. I updated the scripts in the commit below

File: source/water/index.rst (hunk 2 above)
Comment (volker-krey) [reply source/water/index.rst]: I have no opinion on this and even had to look up what the 'Oxford comma' is. Let's take a vote in tomorrow's meeting.

**Comment by volker-krey** [comment]
@scparkinson this looks very good to me. From a structure perspective, it might make sense to split the water section into water for energy, water for land and other uses (industry, residential). This would mirror the structure under emissions where we also have an energy and land part. Any thoughts on this?

**Comment by scparkinson** [comment]
I can definitely add the description of the other water uses but purposefully left out in this case because it was initially indicated just to include the water for energy part. Please let me know if you want me to add the description of the other water uses.

**Comment by volker-krey** [comment]
@scparkinson thanks for the offer, but I would suggest that for the moment we keep it at the level you have provided which is consistent with the mainstream version of the global model version. I just thought that structurally it would make sense to anticipate that other components than "water for energy" will be added at some point.

**Pull request merged**